README.md

```diff
@@ -10,35 +10,25 @@ If you are looking for the loader which uses [EJS templating engine](https://git
 
 ## Usage
 
-[Documentation: Using loaders](http://webpack.github.io/docs/using-loaders.html)
+[Documentation: Using loaders](https://webpack.js.org/concepts/loaders/)
 
 ``` javascript
-var template = require("ejs!./file.ejs");
+var template = require("./file.ejs");
 // => returns the template function compiled with undesrcore (lodash) templating engine.
 
 // And then use it somewhere in your code
 template(data) // Pass object with data
 ```
 
-You also should provide a global `_` variable with the lodash/underscore runtime. You can do it with the following webpack plugin: https://github.com/webpack/docs/wiki/list-of-plugins#provideplugin
```

Comment:
Not sure why this was removed either

Reply:
It didn't work for me because it produced an error as I pasted the readme example that uses underscore instead of lodash in my config.
Also, since my project didn't require any escaping it didn't show any errors.
Fixed.

Reply:
I'm not sure how this is gonna work with `lodash` so please update to the working example